apps/web/screens/UniversitySearch/Comparison.tsx

```diff
@@ -0,0 +1,337 @@
+import React, { useEffect, useState } from 'react'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import {
+  Box,
+  Button,
+  GridContainer,
+  LinkV2,
+  Table as T,
+  Text,
+} from '@island.is/island-ui/core'
+import { Screen } from '@island.is/web/types'
+import {
+  GetNamespaceQuery,
+  GetNamespaceQueryVariables,
+  GetUniversityGatewayByIdQuery,
+  GetUniversityGatewayByIdQueryVariables,
+  GetUniversityGatewayUniversitiesQuery,
+  ProgramDetails,
+  University,
+} from '@island.is/web/graphql/schema'
+import {
+  GET_UNIVERSITY_GATEWAY_PROGRAM,
+  GET_UNIVERSITY_GATEWAY_UNIVERSITIES,
+} from '../queries/UniversityGateway'
+import { GET_NAMESPACE_QUERY } from '../queries'
+import { useLinkResolver, useNamespace } from '@island.is/web/hooks'
+import { TranslationDefaults } from './TranslationDefaults'
+
+import getConfig from 'next/config'
+import { CustomNextError } from '@island.is/web/units/errors'
+
+const { publicRuntimeConfig = {} } = getConfig() ?? {}
+
+interface UniversityComparisonProps {
+  data: Array<ProgramDetails>
+  locale: string
+  namespace: Record<string, string>
+  universities: Array<University>
+}
+
+const Comparison: Screen<UniversityComparisonProps> = ({
+  data,
+  locale,
+  namespace,
+  universities,
+}) => {
+  const n = useNamespace(namespace)
+  const { linkResolver } = useLinkResolver()
+  const [selectedComparison, setSelectedComparison] =
+    useState<Array<ProgramDetails>>(data)
+
+  const handleDelete = (dataItem: ProgramDetails) => {
+    let found = false
+    selectedComparison.forEach((x) => {
+      if (x.id === dataItem.id) {
+        found = true
```

Comment:
I don't know the context for this but depending on how many items there are for comparison it might be more efficient to `break` after found so that this loop stops iterating.

Reply:
Though, IIRC I think foreach is illegal in regards to `break` statements but you could leverage 

```typescript
const found = selectedComparison.some((x) => x.id === dataItem.id)
```

Reply:
The max number of items in comparison is 3 so I don't think this would become a problem. But I can switch to .some anyways :)